Bookshelf/shared/build.gradle.kts

```diff
@@ -30,7 +30,6 @@ kotlin {
     val ktorVersion = "1.6.1"
     val serializationVersion = "1.2.1"
     val coroutinesVersion = "1.5.2-native-mt"
```

Comment:
any reason why this is still set to `1.5.x` it could send the wrong message since we're requiring `1.6.x`

Reply:
I just forgot one ... shuffling through five Intellij and five Xcode instances 🤯 🤪